kf_lib_data_ingest/tag_commit_cache.json

```diff
@@ -0,0 +1,5 @@
+{
+    "0.1.0": "502587d550fdc9ba1a7adc68a836a02b78c0a0f7",
+    "0.2.0": "271f03ff7b0b9477017dd837d9ae4c272cac89e5",
+    "0.3.0": "f1a50035c3d463c78f13abc3b0cb353afe200c0b"
+}
```

Comment:
Does this mean you want to update this file every release?

Reply:
This is now redundant with __version__ = '0.3.0' in __init__.py, since we can look at the largest tag.

Reply:
Bc someone may pip install version 0.2.0 or a version that is not the latest version. You still need to get the commit hash for that version somehow

Reply:
I propose making the version file (\_\_init\_\_.py) always just have both the tag and also the commit hash for the current version and nothing else, get rid of this file, and get rid of the lookup function. If someone installs an older version, their version file would include the appropriate hash for that version. We can work on replacing manual updates of the version file with automated ones in another pr for the release process.